package/base-files/files/lib/upgrade/legacy-sdcard.sh

```diff
@@ -7,12 +7,12 @@ legacy_sdcard_check_image() {
 	return 1
 	}
 
-	get_partitions "/dev/$diskdev" bootdisk
+	get_partitions "/dev/$diskdev" bootdisk || return 1
```

Comment:
There are many other callers of `get_partitions`

Reply:
Oops, I had completely forgotten about that... I'll try to update other callers.